imhotep/app.py

```diff
@@ -307,6 +308,12 @@ def parse_args(args: List[str]) -> argparse.Namespace:
     arg_parser.add_argument(
         "--commit", help="The sha of the commit to run static analysis on."
     )
+    arg_parser.add_argument(
+        "--base_branch_name",
+        required=False,
+        default="main",
```

Comment:
This should be master so it's backwards compatible, imo. Not pressing.

Reply:
Sure.